app/src/main/java/com/github/gogetters/letsgo/activities/ChatActivity.kt

```diff
@@ -75,12 +72,12 @@ class ChatActivity : AppCompatActivity() {
 
         if (text.isNotEmpty()) {
             val fromId = userId!!
-            var toId: String?
-            if (toUser?.id == null) {
-                toId = UNKNOWN
-            } else {
-                toId = toUser?.id
-            }
+            val toId = toUser!!.id!! // This might work?
+//            if (toUser?.id == null) {
```

Comment:
no need to leave commented code, that's what git is for (I do it as well but it's not good practice)